### DISCUSSION TITLE
Suggestion Feed

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#67] [Ideas]
What you need, when you need it. The Feed proactively serves you workflow materials related to your current workflow context, such as your currently opened code files, browser tabs, and more.

**Comment by Aayushpradhan88-js** [comment]
Hello

**Comment by Aayushpradhan88-js** [comment]
console.log("Aayush Pradhan")

**Comment by 1biancheng** [comment]
> 您需要什么，何时需要。源会主动为您提供与当前工作流上下文相关的工作流材料，例如当前打开的代码文件、浏览器选项卡等。

Why there isn't a multi-agent system driving force as pieces?

**Comment by joshuadanpeterson** [comment]
This would be great. I love how intelligent Pieces is, and so this fits perfectly.

**Comment by adams-coding** [comment]
Loving pieces.
Would love to see something like this built into it:
https://github.com/frdel/agent-zero

**Comment by rcesaret** [comment]
Really loving Pieces!!
Do you guys have plans to add Snippet Discovery support for .r and .rmd files anytime soon?

**Reply by mark-at-pieces** [reply]
@sam-at-pieces

**Comment by joshuadanpeterson** [comment]
I already commented on this, but I'm excited for when this is finally released. This will help operationalize the snippets that I've saved. I can't wait.

**Comment by randyhoulahan** [comment]
MCP support - to connect to other servers.  Big deal breaker.  Still using for the long term memory as I let my other chat interfaces connect to your mcp.

Drive is nice, a folder system or project system needed.  Perhaps just changing the UI so that tags can be grouped in display and by reference.  I have not tested if accessable via mcp but I assume no.  Would be nice.

Agent templates =  llm, drive (file, (folder, project, tag) ref to a group), instructions (ref to drive ).

Async chats - maybe it does I have not tested (stoped using chat because no MCP to other servers).

Cron tasks - scheduled prompts.

Task templates - which agent template, or the info from the agent template plus the prompt ... to be ran ... at a schedule time or interval.

OS - menu options like in vs code.  I would like things saved to memory and or drive outside of coding/work.  Browser too, not just code.

Workstream - llm analysis and suggestions.  Efficiency improvement, maybe set some goals, efficiany learning ect.

WOuld love a graph explorer of memory.

Further your MCP we will call long term memory.  There should be a cloud only option.  Local is great for privacy.  Cloud great for convenience.  The option would be nice.

Just my observations thank you.

**Reply by 1biancheng** [reply]
> MCP support - to connect to other servers.  Big deal breaker.  Still using for the long term memory as I let my other chat interfaces connect to your mcp.
> 
> Drive is nice, a folder system or project system needed.  Perhaps just changing the UI so that tags can be grouped in display and by reference.  I have not tested if accessable via mcp but I assume no.  Would be nice.
> 
> Agent templates =  llm, drive (file, (folder, project, tag) ref to a group), instructions (ref to drive ).
> 
> Async chats - maybe it does I have not tested (stoped using chat because no MCP to other servers).
> 
> Cron tasks - scheduled prompts.
> 
> Task templates - which agent template, or the info from the agent template plus the prompt ... to be ran ... at a schedule time or interval.
> 
> OS - menu options like in vs code.  I would like things saved to memory and or drive outside of coding/work.  Browser too, not just code.
> 
> Workstream - llm analysis and suggestions.  Efficiency improvement, maybe set some goals, efficiany learning ect.
> 
> WOuld love a graph explorer of memory.
> 
> Further your MCP we will call long term memory.  There should be a cloud only option.  Local is great for privacy.  Cloud great for convenience.  The option would be nice.
> 
> Just my observations thank you.
> 
> 

No idea — the last official reply I got was two years ago. As a user, I can only say the wish-list that you and I posted just doesn’t seem to fit Pieces’ roadmap; so far there hasn’t been any exciting progress that matches our expectations.
